vmware_video/kernel/device.c

```diff
@@ -332,10 +332,10 @@ ControlHook(void *dev, uint32 msg, void *buf, size_t len)
 		case VMWARE_SET_PALETTE:
 		{
 			uint8 colors[3 * 256];
-			if (user_memcpy(colors, buf, sizeof(colors)) < B_OK)
-				return B_BAD_ADDRESS;
 			uint8 *color = colors;
 			uint32 i;
+			if (user_memcpy(colors, buf, sizeof(colors)) < B_OK)
+				return B_BAD_ADDRESS;
```

Comment:
C89 standard requires declarations to stay at the beginning of a block

Reply:
OK